src/couch_index_server.erl

```diff
@@ -38,7 +38,6 @@
 -record(st, {root_dir}).
 
 start_link() ->
-    couch_epi:register_service(couch_index),
```

Comment:
in the other PR's this is replaced by an addition to the supervision tree, but I don't see that in this PR.